src/hermes.jsx

```diff
@@ -0,0 +1,338 @@
+'use strict';
+import React, {PropTypes} from 'react';
```

Comment:
would prefer `import React, { PropTypes, Component } from 'react';` since we're neck deep in ES6.